Formula/v/vte3.rb

```diff
@@ -35,12 +36,29 @@ class Vte3 < Formula
   depends_on "pango"
   depends_on "pcre2"
 
+  uses_from_macos "python" => :build
+
   on_macos do
     depends_on "llvm" => :build if DevelopmentTools.clang_build_version <= 1500
     depends_on "gettext"
     # Undefined symbols for architecture x86_64:
     #   "std::__1::__libcpp_verbose_abort(char const*, ...)", referenced from: ...
     depends_on "llvm" if DevelopmentTools.clang_build_version <= 1400
+
+    # Use fast_float implementation for from_chars
+    # upstream bug report, https://gitlab.gnome.org/GNOME/vte/-/issues/2823
+    patch do
+      url "https://gitlab.gnome.org/kraj/vte/-/commit/c90b078ecf4382458a9af44d765d710eb46b0453.diff"
+      sha256 "fd8fd85339df1aa5ffb2617d6e67d26e26abb9caeb06ef7766b13341231c2c79"
+    end
+    patch do
+      url "https://gitlab.gnome.org/kraj/vte/-/commit/2a32e43e43b04771a3357d3d4ccbafa7714e0114.diff"
+      sha256 "f69f103b19de93f94fca05dea5a151b4109085ce716472acddb9a112502437d4"
+    end
```

Comment:
Can we use our `fast_float` formula here? We may need to provide the amalgamated header, but that seems fine.

Reply:
Would be good to handle this at some point.